lib/resources/r4/encounter.yaml

```diff
@@ -546,3 +546,92 @@ fields:
         system: http://terminology.hl7.org/CodeSystem/v3-ObservationValue
         info_link: http://hl7.org/fhir/r4/v3/ObservationValue/cs.html
     note: Encounters created with meta.security set with code 'UNCERTREL' will be returned with an external encounter type.
+
+- name: Client Organization Extension
+  required: 'Yes'
+  type: Reference (Organization)
+  description: The financially responsible organization.
+  url: https://fhir.cerner.com/millennium/r4/encounters/encounter/#extensions
+  example: |
+    {
+      "serviceProvider": {
+        "reference": "Organization/675844"
+      }
+    }
+
+- name: Estimated Financial Responsibility Amount Extension
+  terminology_name: extension[x].valueMoney.currency
+  required: 'Yes'
+  type: Money
+  description: Currency codes from ISO 4217 (see https://www.iso.org/iso-4217-currency-codes.html).
+  url: https://fhir.cerner.com/millennium/r4/encounters/encounter/#extensions
```

Comment:
Should all of these urls be 'https://fhir.cerner.com/millennium/r4/encounters/encounter/#extensions'

not 'http://hl7.org/fhir/r4/datatypes-definitions.html#Money'

Reply:
they should be the fhir.cerner.com URL